### PR TITLE
Calvin/tiling and mailes

### PR DESCRIPTION
**Pull request by calvind777**
## Feature Name
Feature Description

### Screenshots

CC: @clairelin135

### CODE REVIEW

**Comment by clairelin135** [comment]
Woooo I'm so excited 😀 

image

One issue--it seems like the column-count property evenly divides the number of elements among the three columns. However, it seems like the elements will display unevenly (like the above)

Can we write a loop in javascript that renders the works images instead? It should keep track of the column heights and add a new image to the shortest column (and the leftmost column if there is a tie).

**Comment by calvind777** [comment]
check new edits

**Comment by clairelin135** [comment]
Thanks for making this fix! That workaround at the end was super smart haha way better than what i thought of!

**Comment by by-co** [comment]
how did u do this without any js ...... this is magic please tech talk and show us ur ways

File: app/assets/stylesheets/works.scss [hunk 1]
```diff
@@ -1,3 +1,19 @@
 // Place all the styles related to the Works controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+.flex-grid {
```
Comment (by-co) [inline app/assets/stylesheets/works.scss]: @calvind777  is this being used anywhere or is this a leftover

**Pull request merged**